qan/analyzer/mysql/worker/perfschema/worker.go

```diff
@@ -331,7 +332,8 @@ func (w *Worker) Status() map[string]string {
 
 func (w *Worker) SetConfig(config pc.QAN) {
```

Comment:
Just in case add mutex to protect access to `w.ticker`.

Reply:
Yes, good point - added.